packages/application-generic/src/usecases/add-job/add-job.command.ts

```diff
@@ -7,6 +7,9 @@ export class AddJobCommand extends EnvironmentWithUserCommand {
   @IsDefined()
   jobId: string;
 
+  @IsDefined()
+  job: JobEntity;
+
   @IsOptional()
-  job?: JobEntity;
+  filtered?: boolean;
```

Comment:
This is the property used to set delay for digest and delay to zero. Name suggestions is more then welcome here :)